src/components/SidebarMain/SidebarMain.js

```diff
@@ -98,7 +109,7 @@ function MainSideBarHeading(props) {
 }
 
 /**
- * Component that renders the SideBarMain component.
+ * Component that renders the SideBarMain component, which displays the usersname and user's role.
```

Comment:
This also displays all the menu items, not only the username and user role.
-> `Sidebar which the user can use to navigate through the mutate environment`